pytorch_lightning/core/lightning.py

```diff
@@ -1718,6 +1718,33 @@ def _set_hparams(self, hp: Union[dict, Namespace, str]) -> None:
         else:
             self._hparams = hp
 
+    def to_onnx(self, file_path: str, input_sample: Optional[Tensor] = None, **kwargs):
+        """Saves the model in ONNX format
+
+        Args:
+            file_path: The path of the file the model should be saved to.
+            input_sample: A sample of an input tensor for tracing.
+            **kwargs: Will be passed to torch.onnx.export function.
+
+        Example:
+            .. code-block:: python
```

Comment:
can we have it rather as doctest?

Reply:
probably not. Can't you just define a model with a single layer or something?

Reply:
Could do that yea 👍